src/abstracts/SablierV2MerkleStreamer.sol

```diff
@@ -86,8 +93,12 @@ abstract contract SablierV2MerkleStreamer is
 
     /// @inheritdoc ISablierV2MerkleStreamer
     function clawback(address to, uint128 amount) external override onlyAdmin {
-        // Checks: the campaign has expired.
-        if (!hasExpired()) {
+        // Safe Interactions: query the protocol fee. This is safe because it's a known Sablier contract that does
+        // not call other unknown contracts.
+        UD60x18 protocolFee = LOCKUP.comptroller().protocolFees(ASSET);
+
+        // Checks: if the protocol fee is not greater than zero, the campaign must be expired.
+        if (!protocolFee.gt(ud(0)) && !hasExpired()) {
             revert Errors.SablierV2MerkleStreamer_CampaignNotExpired({
                 currentTime: block.timestamp,
                 expiration: EXPIRATION
```

Comment:
We need a new error for the case when the protocol fee is not zero.

Reply:
as said below, we shouldn't